scripts/cipublish

```diff
@@ -35,10 +35,13 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
                    "${AWS_ECR_ENDPOINT}/raster-foundry-app-backend:${GIT_COMMIT}"
             docker tag "raster-foundry-airflow:${GIT_COMMIT}" \
                    "${AWS_ECR_ENDPOINT}/raster-foundry-airflow:${GIT_COMMIT}"
+            docker tag "raster-foundry-migrations:${GIT_COMMIT}" \
```

Comment:
This reference needs to match what's in `docker-compose.test.yml`, which is `raster-foundry-app-migrations`.